### PR TITLE
Clarify some documentation in the `common` module

### PR DESCRIPTION
**Pull request by FWDekker**

### CODE REVIEW

File: modules/common/src/main/kotlin/org/cafejojo/schaapi/common/LibraryUsageGraphGenerator.kt [hunk 1]
```diff
@@ -5,7 +5,7 @@ package org.cafejojo.schaapi.common
  */
 interface LibraryUsageGraphGenerator {
     /**
-     * Generates usage graphs for each method in each class of the user project.
+     * Generates all usage graphs for the user project with respect to the library graph.
```
Comment (gandreadis) [inline modules/common/src/main/kotlin/org/cafejojo/schaapi/common/LibraryUsageGraphGenerator.kt]: with respect to the `library project`?

File: modules/common/src/main/kotlin/org/cafejojo/schaapi/common/TestGenerator.kt [hunk 2]
```diff
@@ -3,13 +3,13 @@ package org.cafejojo.schaapi.common
 import java.io.File
 
 /**
- * Represents the test generator that generates tests based on patterns.
+ * Generates test files based on [Pattern]s.
  */
 interface TestGenerator {
     /**
-     * Generates tests based on patterns.
+     * Generates  a test file based on the given [patterns].
```
Comment (gandreadis) [inline modules/common/src/main/kotlin/org/cafejojo/schaapi/common/TestGenerator.kt]: Duplicate space

File: modules/common/src/main/kotlin/org/cafejojo/schaapi/common/LibraryUsageGraphGenerator.kt (hunk 1 above)
Comment (FWDekker) [reply modules/common/src/main/kotlin/org/cafejojo/schaapi/common/LibraryUsageGraphGenerator.kt]: Exactly. Thanks!

File: modules/common/src/main/kotlin/org/cafejojo/schaapi/common/TestGenerator.kt (hunk 2 above)
Comment (FWDekker) [reply modules/common/src/main/kotlin/org/cafejojo/schaapi/common/TestGenerator.kt]: Whoops. Thanks!

**Pull request merged**